### ISSUE TITLE
Error `using` package

### ISSUE THREAD

**Issue by MFairley** [JuliaPOMDP/QuickPOMDPs.jl#16]
I'm getting an error with `using QuickPOMDPs`:

```
julia> using QuickPOMDPs
[ Info: Precompiling QuickPOMDPs [8af83fb2-a731-493c-9049-9e19dbce6165]
ERROR: LoadError: LoadError: UndefVarError: n_states not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at ./Base.jl:26
 [2] top-level scope at /Users/mfairley/.julia/packages/QuickPOMDPs/rucZo/src/discrete_explicit.jl:31
 [3] include(::Module, ::String) at ./Base.jl:377
 [4] include(::String) at /Users/mfairley/.julia/packages/QuickPOMDPs/rucZo/src/QuickPOMDPs.jl:1
 [5] top-level scope at /Users/mfairley/.julia/packages/QuickPOMDPs/rucZo/src/QuickPOMDPs.jl:12
 [6] include(::Module, ::String) at ./Base.jl:377
 [7] top-level scope at none:2
 [8] eval at ./boot.jl:331 [inlined]
 [9] eval(::Expr) at ./client.jl:449
 [10] top-level scope at ./none:3
in expression starting at /Users/mfairley/.julia/packages/QuickPOMDPs/rucZo/src/discrete_explicit.jl:31
in expression starting at /Users/mfairley/.julia/packages/QuickPOMDPs/rucZo/src/QuickPOMDPs.jl:12
```

**Comment by zsunberg** [comment]
Hi @MFairley thanks for reporting! This is because we have not upgraded this package to be compatibile with POMDPs 0.9 yet. If you do `pkg> add POMDPs@v0.8`, it should work. In the mean time we will upgrade to work with 0.9

**Comment by zsunberg** [comment]
This should be fixed in QuickPOMDPs v0.2.5. Unfortunately it might be a few weeks before we get all the solvers up to date with POMDPs v0.9. Sorry about the roughness!
